src/main/resources/db/migration/V20221214201500__Create_Initial_Tables.sql

```diff
@@ -4,5 +4,6 @@ create table events
     contract_id varchar not null,
     event_id    varchar not null,
     currency    varchar not null,
-    amount      numeric not null
+    amount      numeric not null,
+    json_data   json
```

Comment:
@fayi-da @chunlokling-da would be interesting (as an example) to add a GIN index on observers in json_data, so something like `where json_data->'observers' ? 'does-this-observer-exist'` and get the index used?

Reply:
pushed change to add the gin index and also API to get events filtered by observers field in the json column

Reply:
Also tested to make sure the new index is being used when doing `where json_data->'observers' ? 'does-this-observer-exist'`

```
SET enable_seqscan TO off;
EXPLAIN select * from events where json_data->'observers' ? 'EUR_Bank::12201f8d3d6fe6285cc964ac20fdceea91f57c0742abf3ed452e88c0897cb9e3cbfd';
```

```
Bitmap Heap Scan on events  (cost=12.03..20.50 rows=4 width=160)
  Recheck Cond: ((json_data -> 'observers'::text) ? 'EUR_Bank::12201f8d3d6fe6285cc964ac20fdceea91f57c0742abf3ed452e88c0897cb9e3cbfd'::text)
  ->  Bitmap Index Scan on json_data_observers_gin  (cost=0.00..12.03 rows=4 width=0)
        Index Cond: ((json_data -> 'observers'::text) ? 'EUR_Bank::12201f8d3d6fe6285cc964ac20fdceea91f57c0742abf3ed452e88c0897cb9e3cbfd'::text)

```